pkg/cmd/server/start/start_master.go

```diff
@@ -245,6 +253,8 @@ func (o MasterOptions) RunMaster() error {
 		return nil
 	}
 
+	// Inject feature flags.
+	masterConfig.FeatureConfig = o.FeatureConfig
```

Comment:
Right. Good idea. It came to my mind as well but in the end I wanted to avoid string-based comparison. Pros overweight the cons in this case though.